scout/adapter/mongo/case.py

```diff
@@ -501,15 +501,15 @@ def delete_case(self, case_id=None, institute_id=None, display_name=None):
         result = self.case_collection.delete_one(query)
         return result
 
-    def load_case(self, config_data, update=False):
+    def load_case(self, config_data, update=False, keep_actions=True):
```

Comment:
Ah, good! My first reading was that the load case -u would end up in adapter.update_case via scout.load_case rather than adapter.load_case, but I got that wrong! Thanks!